src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -67,6 +94,11 @@ const MainNavDroplist = Droplist.extend({
   onSelect({ model }) {
     model.get('onSelect')();
   },
+  initialize({ lists, selected }) {
+    this.lists = lists;
```

Comment:
lists is attached by default so it doesn't need to happen here.

Also the `selected` can be passed in on a `state` object as well as in this example below, so the `initialize` isn't necessary

```
    const sortSelect = new SortDroplist({
      collection: this.sortOptions,
      state: { selected: this.sortOptions.get(this.getState().getSort()) },
    });

```